src/status_im/ui/screens/wallet/events.cljs

```diff
@@ -96,6 +98,50 @@
  (fn [{:keys [web3 obj success-event]}]
    (ethereum/estimate-gas-web3 web3 (clj->js obj) #(re-frame/dispatch [success-event %2]))))
 
+(defn- validate-token-name! [web3 {:keys [address symbol name]}]
```

Comment:
do we have tests for these functions?

Reply:
Would they be useful? If I mock up web3 the only thing that remains to be tested is equality check?

Reply:
ok